web/src/containers/activity/CostAllocateFFP.test.js

```diff
@@ -153,13 +148,14 @@ describe('the CostAllocateFFP component', () => {
           year: '1067'
         }
       ],
-      costAllocation: state.activities.byKey.key.costAllocation
+      costAllocation: state.apd.data.activities[0].costAllocation
```

Comment:
This works because the object has a property with key `0`. It should just keep working when the state is updated to an array, because instead of `0` being a property key, it'll be an array index.